pid_controller/src/pid_controller.cpp

```diff
@@ -26,6 +26,9 @@
 #include "control_msgs/msg/single_dof_state.hpp"
 #include "controller_interface/helpers.hpp"
 
+#include <rclcpp/rclcpp.hpp>
+#include <rclcpp/version.h>
```

Comment:
```suggestion
#include "rclcpp/rclcpp.hpp"
#include "rclcpp/version.h"
```
Thanks for the heads-up on the version header :)